src/romt/crate.py

```diff
@@ -77,7 +77,7 @@ def crate_prefix_from_name(name: str) -> str:
         prefix = "3/{}".format(name[0])
     else:
         prefix = "{}/{}".format(name[:2], name[2:4])
```

Comment:
Side note, since python 3.6, f-strings are recommended over `.format` and `%`.
This will be more performant and more readable:
```suggestion
        prefix = f"{name[:2]}/{name[2:4]}"
```